xbmc/rendering/RenderSystem.cpp

```diff
@@ -90,10 +93,7 @@ void CRenderSystemBase::ShowSplash(const std::string& message)
 
   if (!m_splashImage)
   {
-    std::string splashImage = "special://home/media/Splash.png";
-    if (!XFILE::CFile::Exists(splashImage))
-      splashImage = "special://xbmc/media/Splash.png";
-
+    std::string splashImage {CUtil::GetSplashPath()};
```

Comment:
What's the reason for the curly braces? why not `std::string splashImage = CUtil::GetSplashPath()`?

Reply:
Matter of taste. I don't really care tbh.

Reply:
please be consistent because you use `=` in `IOSExternalTouchController.mm`

Reply:
Yeah I went for minimal changes there because it was only one line, but in principle you're of course right. I think it would be good to have some of this stuff defined in the coding style guidelines maybe?